cross-platform/react-app/src/frontend/BottomPanels/ToolsBottomPanel.tsx

```diff
@@ -24,13 +24,13 @@ import "./ToolsBottomPanel.scss";
 
 export type ButtonRowProps = React.HTMLAttributes<HTMLDivElement>;
 
+/** Information about one entry in the Tools panel. */
 export interface ToolEntry {
   labelKey: string;
   icon?: string;
   toolItemDef: ToolItemDef;
 }
 
-// tslint:disable-next-line: variable-name
 export const ButtonRow = React.forwardRef((props: ButtonRowProps, ref: MutableHtmlDivRefOrFunction) => {
```

Comment:
@toddsouthenbentley The undocumented components here are undocumented because I don't fully understand them. Since this is relatively advanced React code, could you document them such that users will understand why they are the way they are?

Reply:
I added some documentation. The forwardRef function is used to surface a ref to the scrollable div to the parent components.